plugins/package-managers/node/src/main/kotlin/Pnpm.kt

```diff
@@ -70,7 +70,7 @@ class Pnpm(
 
     override fun command(workingDir: File?) = if (Os.isWindows) "pnpm.cmd" else "pnpm"
 
-    override fun getVersionRequirement(): RangesList = RangesListFactory.create("5.* - 7.*")
+    override fun getVersionRequirement(): RangesList = RangesListFactory.create("5.* - 8.*")
```

Comment:
Would it make sense to bump `PNPM_VERSION` in docker in this PR, so that we have it covered by the tests?